release-notes.txt

```diff
@@ -0,0 +1,241 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-15
+
+! New in this release
+- baseboxd:
+  - updated version to 1.11.1
+  - added initial support for Broadcom KNET interfaces
```

Comment:
Since this is now basically duplicated in the experimental features, we might was well drop this and compress this to one line:
```suggestion
- baseboxd: updated to version 1.11.1
```